resources/views/partials/login.blade.php

```diff
@@ -0,0 +1,4 @@
+<x-rapidez::button :href="route('account.login')" dusk="product-alert-login" :attributes="$login->attributes->class('flex items-center justify-center w-full')">
```

Comment:
Can't we do something like this?
```suggestion
<x-rapidez::button {{ $login->attributes->class('flex items-center justify-center w-full')->merge([
    'href' => route('account.login'),
    'dusk' => 'product-alert-login'.
]) }}>
```
Or this?
```suggestion
<x-rapidez::button {{ $login->attributes->merge([
    'class' => 'flex items-center justify-center w-full',
    'href' => route('account.login'),
    'dusk' => 'product-alert-login'.
]) }}>
```

Reply:
Yes! i've added all the attributes in the merge!

Reply:
Nice, but is `:attributes` needed?